.github/workflows/build.yaml

```diff
@@ -0,0 +1,51 @@
+name: Build
+
+on:
+- pull_request
+- push
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install build dependencies
+        run: |-
+          sudo apt-get update
+          sudo apt-get install --yes --no-install-recommends \
+            bison \
+            doxygen \
+            flex \
+            gettext \
+            graphviz \
+            liborc-0.4-dev \
+            libpng-dev \
+            libasound2-dev \
+            libsdl1.2-dev \
+            libgl1-mesa-dev \
+            pkg-config
+
+      - name: Checkout Git branch
+        uses: actions/checkout@v2.0.0
+
+      - name: Run CMake
+        run: |-
+          mkdir build
+          cd build
+          cmake ../libvisual/ \
+            -DENABLE_ALSA=yes \
+            -DENABLE_DANCINGPARTICLES=yes \
+            -DENABLE_DOCS=yes \
+            -DENABLE_GOOM2K4=yes \
+            -DENABLE_NLS=yes
+
+      - name: Run "make"
+        run: |-
+          cd build
+          make VERBOSE=1 -j2
```

Comment:
You could combine the two with `make -C build -j2 VERBOSE=1` to shorten the lines

Reply:
True, why not.